lib/rubocop/cop/style/redundant_current_directory_path.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for uses a redundant current directory path in `require_relative`.
```

Comment:
Maybe we can make this a configurable list of method names from the beginning?

Reply:
Good point. This bothered me. Published configuration options must be stable and cannot be renamed later.
So I thought I could expose it as a configuration option when needed.

Of course it can also be made configurable from the beginning. Do you prefer it?

Reply:
Seems to me a name like `TargetMethods` (or similar) would work fairly well. I made my remark just because you said you wanted to keep the name of the cop generic in case more methods are added down the road. I assume that there are quite a few methods that operate on the current directory (but we'll have to double check this), so this makes sense to me. 

Anyways, not a big deal for me - just a small point of consideration.

Reply:
`TargetMethods` looks good to me! On the other hand, I thought about it again. The redundant `./` location might be not always the first argument, like `require_relative './path/to/feature`.

So I thought it would be better if the configuration will be extended based on the feedback. What do you think?

Reply:
Sure.